openhands/agenthub/codeact_agent/codeact_agent.py

```diff
@@ -184,6 +187,14 @@ def get_action_message(
                     content=content,
                 )
             ]
+        elif isinstance(action, CmdRunAction) and action.source == 'user':
+            content = [TextContent(text=action_to_str(action))]
```

Comment:
I would simply do this:

```suggestion
            content = [TextContent(text=f'User executed the command:\n{action.command}')]
```
Then get rid of action_to_str, since we only use it once here - WDYT?